spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -144,13 +144,30 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
   def toJTensor(tensor: Tensor[T]): JTensor = {
     // clone here in case the the size of storage larger then the size of tensor.
     require(tensor != null, "tensor cannot be null")
-    if (tensor.nElement() == 0) {
-      JTensor(Array(), Array(0), bigdlType = typeName)
-    } else {
-      val cloneTensor = tensor.clone()
-      val result = JTensor(cloneTensor.storage().array().map(i => ev.toType[Float](i)),
-        cloneTensor.size(), bigdlType = typeName)
-      result
+    tensor.getTensorType match {
+      case SparseType =>
+        // Note: as SparseTensor's indices is inaccessible here,
+        // so we will transfer it to DenseTensor. Just for testing.
+        if (tensor.nElement() == 0) {
```

Comment:
return back a __sparse tensor__ should be supported here instead of return a dense one for passing around the unitest.